resources/type-system/subtyping.md

```diff
@@ -121,12 +134,31 @@ We say that a type `T0` is a subtype of a type `T1` (written `T0 <: T1`) when:
 
 - **Right Top**: `T1` is a top type (i.e. `Object`, `dynamic`, or `void`).
 
-- **Left Bottom**: `T0` is `Null`
+- **Left Bottom**: `T0` is `Never`
+
+- **Right Legacy Top**: `T1` is a legacy top type (i.e. `Object*`, `dynamic*`,
+  or `void*`).
+
+- **Left Legacy Null** `T0` is `Null*`
+
+- **Left and Right Legacy** `T0` is `S0*` and `T1` is `S1*` and `S0 <: S1`.
```

Comment:
This would not allow us to conclude `List<Null*> <: List<int*>`, because that will require `Null <: int`. Wouldn't that break too much existing code?

(Left Legacy and Right Legacy can't do it, either.)

Reply:
Yes, agreed, I realized that my treatment of `T*` in this CL is completely broken, I have revised it from first principles in a follow on.  Will send out shortly.